CMakeLists.txt

```diff
@@ -11,6 +11,10 @@ set(CMAKE_MODULE_PATH
 
 include(cmake/utils.cmake)
 
+# Set CFLAGS/CXXFLAGS
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Werror -std=gnu11")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Werror -std=gnu++11")
```

Comment:
Ther is no C++ code here.

Reply:
Removed

Reply:
Now, given @Totktonada has pointed that out, I start to worry about this option too. We do not have C++ code there (yet) and there is no point to enforce [awkward GNU extensions for C++](https://gcc.gnu.org/onlinedocs/gcc/C_002b_002b-Extensions.html) language at the moment when it's unnecessary.

Reply:
Sorry for the cofusion created - have not updated browser before sending comment. Now I see it's fixed